<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatTools</class>
 <widget class="QWidget" name="FloatTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{background-color: rgb(225, 225, 225);border:1px groove gray;border-radius:4px;padding:1px 4px;border-style: outset;}
QToolButton:hover{background-color:rgb(229, 241, 251); color: black;}
QToolButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}

QMenu {
    background-color : rgb(253,253,254);
 padding:5px;
 border-radius:15px;
}
QMenu::item {
    font-size:10pt;
    color: rgb(0,0,0);
    background-color:rgb(253,253,254);
    padding: 8px 25px 6px 10px;
    margin: 4px 1px;
}
QMenu::item:selected {
    background-color : rgb(236,236,237);
}
QMenu::icon:checked {
    background: rgb(253,253,254);
    position: absolute;
    top: 1px;
    right: 1px;
    bottom: 1px;
    left: 1px;
}
QMenu::icon:checked:selected {
    background-color : rgb(236,236,237);
    background-image: url(:/space_selected.png);
}
QMenu::separator {
    height: 2px;
    background: rgb(235,235,236);
    margin-left: 10px;
    margin-right: 10px;
}
</string>
  </property>
  <widget class="QToolButton" name="toolButtonFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>文件</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/file.png</normaloff>
     <normalon>:/Icons/file.png</normalon>:/Icons/file.png</iconset>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/view.png</normaloff>
     <normalon>:/Icons/view.png</normalon>:/Icons/view.png</iconset>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/edit.png</normaloff>
     <normalon>:/Icons/edit.png</normalon>:/Icons/edit.png</iconset>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/import.png</normaloff>:/Icons/import.png</iconset>
   </property>
   <property name="text">
    <string>导入模型</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/export.png</normaloff>:/Icons/export.png</iconset>
   </property>
   <property name="text">
    <string>导出模型</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/reset.png</normaloff>:/Icons/reset.png</iconset>
   </property>
   <property name="text">
    <string>视角重置</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/Select.png</normaloff>:/Icons/Select.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actiondeleteFace">
   <property name="icon">
    <iconset resource="../../Resources/resources.qrc">
     <normaloff>:/Icons/deleteface.png</normaloff>:/Icons/deleteface.png</iconset>
   </property>
   <property name="text">
    <string>删除面</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionInvertSelect">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionLinkFace">
   <property name="text">
    <string>链接选择</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionClearSelect">
   <property name="text">
    <string>清除选择</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
